### PR TITLE
Add field with date of creation

### PR DESCRIPTION
**Pull request by Natim**
Refs: #39

### CODE REVIEW

**Comment by leplatrem** [comment]
Shouldn't we wait to have «auto-field» ?

**Comment by Natim** [comment]
Within the scope of xml2kinto it is not suposed to be an auto-field

File: amo-blocklist.json [hunk 1]
```diff
@@ -157,7 +157,8 @@
               "name": {"type": "string"},
               "why": {"type": "string"},
               "who": {"type": "string"},
-              "bug": {"type": "string"}
+              "bug": {"type": "string"},
+              "created": {"type": "date-time"}
```
Comment (Natim) [inline amo-blocklist.json]: We probably want to add it for certificates too once #75 too.

**Comment by Natim** [comment]
> Shouldn't we wait to have «auto-field» ?

However we might want to create a new issue to ask for this feature that will be must useful on the kinto-admin

**Comment by Natim** [comment]
Does this works for you @leplatrem ?

**Comment by leplatrem** [comment]
Yes, we can keep #39 opened or open another for «auto field»

**Pull request merged**